mkosi/__init__.py

```diff
@@ -7715,12 +7715,20 @@ def qemu_check_kvm_support() -> bool:
 def run_qemu_cmdline(args: MkosiArgs) -> Iterator[List[str]]:
     accel = "kvm" if args.qemu_kvm else "tcg"
 
+    if "uefi" in args.boot_protocols:
+        mode = "uefi"
+    elif "bios" in args.boot_protocols:
+        mode = "bios"
+    else:
+        mode = "uefi"
```

Comment:
```suggestion
    mode = "bios" if args.boot_protocols == ["bios"] else "uefi"
```
We ensure early on that `args.boot_protocols` can at most be `["bios", "uefi"]` (in some order).

Reply:
Yeah but when we add the linux boot we'll need to split it up again so I would keep it as is